documentation/Experian Commerce Cloud Integration Manual.md

```diff
@@ -377,4 +372,4 @@ Checkout Form | ✔ | ✔ | ✔ | ✔
 * Timeout connection issues can be experience due to a slow internet connection. If you don’t get an answer in the default time; you can try making the request again.
 
 ##  7\. Failover and recovery
-* If the unicorn/pegasus libraries connection fails the store will continue with it's normal workflow; this issue may be caused due a non valid token or slow internet connection.
\ No newline at end of file
+* If the unicorn/pegasus libraries connection fails the store will continue with it's normal workflow; this issue may be caused due a non valid token or slow internet connection.
```

Comment:
Since Unicorn/Pegasus is a code name let's just change this to say _Experian data quality libraries_